src/custom-components/customComponentsTypes.ts

```diff
@@ -0,0 +1,29 @@
+export type CustomComponentType = 'CC' | 'CC2'
+
+export interface CComponent {
```

Comment:
Why do we need so many custom types, is it just because of `CustomComponentType`

Reply:
> Why do we need so many custom types, is it just because of `CustomComponentType`

Not needed now, removed!